modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/store/Store.es.js

```diff
@@ -0,0 +1,256 @@
+/**
```

Comment:
Most of this is copy/paste from @wincent's example except for a few areas that I note below